lib/srv/reexec.go

```diff
@@ -1430,3 +1467,37 @@ func (o *osWrapper) newParker(ctx context.Context, credential syscall.Credential
 
 	return nil
 }
+
+func initLogger(name string, cfg ExecLogConfig) {
+	logWriter := os.NewFile(LogFile, fdName(LogFile))
+	if logWriter == nil {
+		return
+	}
+
+	fields, err := logutils.ValidateFields(cfg.ExtraFields)
+	if err != nil {
+		return
+	}
+
+	var logger *slog.Logger
+	switch cfg.Format {
+	case "":
+		fallthrough // not set. defaults to 'text'
+	case "text":
```

Comment:
Suggestion: combine these two cases to avoid the fallthrough
```suggestion
	case "text", "":
```